orchestra/models/core/models.py

```diff
@@ -64,13 +65,16 @@ class WorkflowVersion(WorkflowVersionMixin, models.Model):
             A longer description of the workflow version.
         workflow (orchestra.models.Workflow):
             The workflow that this is a version of.
+        sanity_checks (str):
+            A JSON blob used to define the sanity checks we run.
```

Comment:
Where do I find an example of the format of this blob?

Reply:
reasonable question, updated docstring